controller/api/models.py

```diff
@@ -251,9 +249,13 @@ def restart(self, **kwargs):
 
     def _clean_app_logs(self):
         """Delete application logs stored by the logger component"""
-        path = os.path.join(settings.DEIS_LOG_DIR, self.id + '.log')
-        if os.path.exists(path):
-            os.remove(path)
+        try:
+            url = 'http://{}:{}/{}/'.format(settings.LOGGER_HOST, settings.LOGGER_PORT, self.id)
+            requests.delete(url)
+        except Exception:
```

Comment:
In general, I agree with the catch what you can handle philosophy, but at this particular spot in the code, there are only two lines within the `try` block and they both have only do to with deleting application logs.  No matter what goes wrong here, I specifically do not want Django handling it, because I don't want the CLI to receive a 500 when a delete request against an app resource succeeds except for the deletion of existing app logs.  Does that make any sense?  Disagreement is welcome.  :smile: